<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="96676" started-at="2017-04-26T06:17:41Z" finished-at="2017-04-26T06:19:18Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="96676" started-at="2017-04-26T06:17:41Z" finished-at="2017-04-26T06:19:18Z">
      <class name="com.disney.alm.test.DemoTest">
        <test-method status="PASS" signature="openBrowser()[pri:0, instance:com.disney.alm.test.DemoTest@111c2a7]" name="openBrowser" is-config="true" duration-ms="11431" started-at="2017-04-26T11:47:41Z" finished-at="2017-04-26T11:47:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="FAIL" signature="iD_12918_loginTest()[pri:0, instance:com.disney.alm.test.DemoTest@111c2a7]" name="iD_12918_loginTest" duration-ms="74115" started-at="2017-04-26T11:47:53Z" finished-at="2017-04-26T11:49:07Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at com.disney.alm.test.DemoTest.iD_12918_loginTest(DemoTest.java:79)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- iD_12918_loginTest -->
        <test-method status="PASS" signature="updateResult()[pri:0, instance:com.disney.alm.test.DemoTest@111c2a7]" name="updateResult" is-config="true" duration-ms="11115" started-at="2017-04-26T11:49:07Z" finished-at="2017-04-26T11:49:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateResult -->
      </class> <!-- com.disney.alm.test.DemoTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
